Ticket4034 motor details de energise

ThomasLohnert opened the pull request:
### Description of work

Adds widget to the motor details OPI to read back/control the on/off status of the motor for Galils.

### Ticket

https://github.com/ISISComputingGroup/IBEX/issues/4034

### Acceptance criteria

- [x] The LED correctly indicates the current motor status
- [ ] The set on/off buttons correctly set the motor status
- [x] The auto-on/off switch correctly applies to the motor
- [x] If the motor does not provide the on/off status PV (i.e. is not a Galil), the widget group is not visible.

### Unit tests

/

### System tests

/

### Documentation

/

---

#### Code Review

- [x] Is the code of an acceptable quality?
- [ ] Do the changes function as described and is it robust?
- [x] Have the changes been documented in the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev)?

### Final Steps
- [ ] Reviewer has moved the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev) entry for this ticket in the "Changes merged into master" section

Commit:
Add controls for on/off

Commit:
tweaks

Commit:
Hide on/off status if not present (=not galil motor)

Commit:
Merge remote-tracking branch 'remotes/origin/master' into Ticket4034_motor_details_de_energise

aaron-long commented:
- The `checkOnOffPresent.py` script isn't really needed and should be implemented as a Rule on the grouping container.
- The `set` label is slightly unclear. Can this be changed to be more descriptive.
- Can you document the de-energise feature, it's PV's, and the behaviour that can result for having it On/Off on the Galil Wiki.

Commit:
Relabel widgets

Commit:
Revert rule to script

ThomasLohnert commented:
- Couldn't get the visibility to work as a rule. Putting it in as a script has an option "execute anyway even if PVs are disconnected" which is missing for rules. Thus the whole container just appears as "disconnected" rather than being hidden.

- Changed labels

- Added Doc [here](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Motors-Trouble-Shooting).
